CONTRIBUTING.md

```diff
@@ -1,19 +1,24 @@
+[中文版](./CONTRIBUTING_cn.md)
+
 # FlagGems Contributor's Guide
 
 Thank you for your interest in FlagGems! We use github to host code, manage issues and pull requests. Before contributing, please read the following guidelines.
 
-## Bug Reports
+## 1. Bug Reports
 Please report bugs using Github's issues. When reporting bugs, please provide
 
 - a brief summary,
 - steps to reproduce,
 - and be specific!
 - Some sample codes will be helpful too.
 
-## Code Contribution
+## 2. Code Contribution
 In pull requests, contributor should describe what changed and why. Please also provide test cases if applicable.
-Pull requests require approvals from two members before merging.
+Pull requests require approvals from __two members__ before merging. Additionally, they must pass continuous integration checks.
```

Comment:
now one member is enough. I forgot to fix it.

Reply:
it's ok, we can fix it any following PR